src/content/components/JiraView/JiraView.js

```diff
@@ -0,0 +1,201 @@
+    computedPointsA = 0;
+  }
+  if (isNaN(computedPointsB)) {
+    computedPointsB = 0;
+  }
+  return computedPointsB - computedPointsA;
+}
+
+function comparePriority(a, b) {
+  let computedPriorityA = a;
+  let computedPriorityB = b;
+  if (!a.startsWith("P")) {
+    computedPriorityA = "P6";
+  }
+  if (!b.startsWith("P")) {
+    computedPriorityB = "P6";
+  }
+  return computedPriorityA.localeCompare(computedPriorityB);
+}
+
+export class JiraView extends React.PureComponent {
+  static contextType = GlobalContext;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loaded: false,
+      awaitingNetwork: false,
+      bugsByEngineer: {},
+      showUnsized: false,
+    };
+  }
+
+  async componentWillMount() {
+    this._isMounted = true;
+    await this.context.qm.runCachedQueries(
+      [
+        {
+          include_fields: includeFields,
+          resolution: "---",
+          order: "cf_fx_points DESC, priority, changeddate DESC",
+          rules: [
+            { key: "keywords", operator: "notsubstring", value: "meta" },
+            {
+              operator: "OR",
+              rules: [
+                {
+                  key: "blocked",
+                  operator: "anywordssubstr",
+                  value: this.context.metas.map(m => m.id).join(","),
+                },
+                {
+                  key: "component",
+                  operator: "anyexact",
+                  value: BUGZILLA_TRIAGE_COMPONENTS.join(","),
+                },
+              ],
+            },
+            {
+              key: "cf_fx_iteration",
+              operator: "notequals",
+              value: "---",
+            },
+            {
+              key: "see_also",
+              operator: "substring",
+              value: "mozilla-hub.atlassian.net/browse/OMC-",
+            },
+          ],
+        },
+      ],
+      () => this._isMounted,
+      ({ rsp: [{ bugs }], awaitingNetwork }) => {
+        let allJiraTickets = {};
+        for (let bug of bugs) {
+          // take bugs from the current release
+          const bugRelease = bug.cf_fx_iteration.split(".")[0];
+          if (
+            parseInt(bugRelease, 10) > this.props.release ||
+            parseInt(bugRelease, 10) < this.props.release - 1
+          ) {
+            continue;
+          }
+          if (bug.see_also) {
+            for (const url of bug.see_also) {
+              let ticket = url.match(
+                /mozilla-hub.atlassian.net\/browse\/(OMC-\d+)/
+              );
+              if (ticket) {
+                let jiraTicket = ticket[1];
+                allJiraTickets[jiraTicket] = allJiraTickets[jiraTicket] || [];
+                allJiraTickets[jiraTicket].push(bug);
+              }
+            }
+          }
+        }
+
+        for (const ticket of Object.keys(allJiraTickets)) {
+          if (!allJiraTickets[ticket].length) {
+            delete allJiraTickets[ticket];
+            continue;
+          }
```

Comment:
I think this isn't necessary for JiraView, since you only add a ticket to allJiraTickets `if (ticket)`. It was only necessary for AllocationView because we added engineers to the list by copying from `teams.omc` rather than by looping over bugs.